cmd/sentry/options/options.go

```diff
@@ -81,6 +82,7 @@ func New(origArgs []string) *Options {
 	fs.StringVar(&opts.ListenAddress, "listen-address", "", "The listen address for the sentry server")
 	fs.IntVar(&opts.HealthzPort, "healthz-port", 8080, "The port for the healthz server to listen on")
 	fs.StringVar(&opts.HealthzListenAddress, "healthz-listen-address", "", "The listening address for the healthz server")
+	fs.BoolVar(&opts.SelfHostedCA, "self-hosted-ca", false, "Runs the CA in self-hosted mode, ie. only arguments provided through command line are considered")
```

Comment:
Please can we use the same “mode” option used for other services?

Reply:
We should favour consistency over the confusing here (though I don’t think it’s super confusing- it’s self hosting your own PKI vs using kubernetes resources as the backend).

Reply:
updated, now takes `--mode`, defaults to `KuberrnetesMode` to keep existing behaviour.

Reply:
Default mode should be self hosted else this breaks container environments